### ISSUE TITLE
Unskilled Attack Roll Target Number Dropdown Only Shows Standard Difficulties

### ISSUE THREAD

**Issue by pemerson71** [MetaMorphic-Digital/torgeternity#237]
One of my Players, tonight, who has no Adds in Unarmed Combat on her Character, attempted to roll to-hit a target, but the standard list, including Target's Dodge or Dexterity, did not show up as a choice.  Instead, the target number table choices showed up, instead [Easy (8), Standard (10), Challenging (12), Hard (14), etc.], for example.  I know that, if we roll an Unskilled (no Adds) skills, it does not re-roll on 10s, only on 20s, so it's important to roll a skill -as opposed to the attribute- to help the system determine any dice explosions, so would it be possible to have the basic combat skills all use the same drop down as skills with Adds in them, please? :)

**Comment by Durak-Git** [comment]
Hello, Is it possible the player forgot to target ?
The description you give is what I can "reproduct" in this case (only the simple DN 6/8/10/12/...)
I have the complete list with a target selected, either with adds or not. (V10 Torg 2.7.1)

**Comment by pemerson71** [comment]
Yes, the Player did target the token they were trying to hit.

I went through and was able to reproduce, on my side, the same thing.

But, I think I might have figured it out... somehow, the token had lost its link to the Threat sheet.  I actually had this happen with a bunch of other tokens, but I had removed the threat sheets for those tokens to the compendium I've been keeping.  This one token had NOT had its threat sheet moved, but when I tried to hit that token, a yellow bar came up saying the threat sheet had been removed and the token... I can't remember, but I think it was the equivalent of the token was no longer active.

I have all of my tokens re-associated with their threat sheets, now, including re-associating that one token, so we'll see if there are any other errors this coming weekend.  If I get a chance, I'll go back in our video and see if I can cut it down and make the video available.  It's super-weird to me, because I was able to do it in version 9, where I could associate a token with its threat sheet, as a default, and moved it into the compendium and not had a problem.  I'll try to do that.
